README.md

```diff
@@ -44,6 +45,11 @@ Some links relevant to users and developers of WTMO:
   [WTMO Developer Guide](https://mana.mozilla.org/wiki/display/DOPS/WTMO+Developer+Guide)
   (behind SSO)
 
+## Contributing to this Repo
```

Comment:
Thanks for updating the README! I think we should clarify that this section applies to contributors without write access to push direct branches to the repo. For contributors with write access, a direct branch is perfectly acceptable.

Reply:
I left the title for the section but clarified the forked PR is only necessary for those without write access.  Do you think that is sufficient?

Reply:
I think the changes are great. Might be good to include a note about contributors with write access -- added a suggestion inline below.